profile/attribute/image_attribute.go

```diff
@@ -11,7 +11,7 @@ import (
 // ImageAttribute is a Yoti attribute which returns an image as its value
 type ImageAttribute struct {
 	attributeDetails
-	value *media.Image
+	value media.Media
```

Comment:
This approach does seem neater, **but** the thing I'm not keen on is having `ImageAttribute.value` and `ImageSliceAttribute.value` return media.Media, rather than media.Image. 

On the one hand, I think this stems from us calling them `Images` in every other SDK, and it's nice to be consistent.

On the other hand, is having `Images` needlessly adding complexity?

@davidgrayston what do you think?

Reply:
In all honesty, I would also like to refactor a bit around how attribute values are done, but I felt it was outside the scope of what I was trying to show here.

Reply:
The way I look at is that what you call Image is essentially one of two different things, a PNG or JPEG. I went with an approach with have these two as distinct types, both fulfilling an interface (media.Media). The validation is done when creating the ImageAttribute structure (and would be reason to not export the `value` variable so integrators can't override it with a non-checked value).